apps/chat/src/store/share/share.epics.ts

```diff
@@ -252,8 +252,13 @@ const sharePromptFolderEpic: AppEpic = (action$) =>
 const shareFailEpic: AppEpic = (action$) =>
   action$.pipe(
     filter(ShareActions.shareFail.match),
-    map(() => {
-      return UIActions.showErrorToast(translate(errorsMessages.shareFailed));
+    map(({ payload }) => {
+      const msg =
+        payload?.toLowerCase()?.trim() === 'bad request'
```

Comment:
it's not good that we doesn't have specific error for sharing of public attachment. Only common `bad request`. Need to connect to Core-team to fix it